docs/modules/ROOT/pages/testing.adoc

```diff
@@ -13,7 +13,6 @@ To use it in your tests, you must add it as a test dependency.
 The following example shows how to add `reactor-test` as a dependency in Maven:
 
 .reactor-test in Maven, in `<dependencies>`
```

Comment:
I don't know whether or how this can be fixed but please verify ALL the `Example N. Title` places in the documentation - some of them now start with the word "Example", some don't. In the current documentation all examples have the same prefix and a significant and useful number. It would be a good idea to keep that.

Reply:
The other problem is that the rendering of these "Example" headers don't provide a background for the monospace font and it is inconsistent with what we had.

Reply:
> I don't know whether or how this can be fixed but please verify ALL the Example N. Title places in the documentation - some of them now start with the word "Example", some don't. In the current documentation all examples have the same prefix and a significant and useful number. It would be a good idea to keep that.

The problem is that before, all  example titles were  indeed **automatically** prefixed with `Example N`, but I cannot find a way to re-enable this behavior.

So, before, we had few titles starting with `Example` word, like these ones:

```
.Example of Callback Hell
.Example of Reactor code equivalent to callback code
.Example of Reactor code with timeout and fallback
.Example of `CompletableFuture` combination
.Example of Reactor code equivalent to future code
.Example of state-based `generate`
```

which were respectively rendered like this:

```
Example 5. Example of Callback Hell
Example 6. Example of Reactor code equivalent to callback code
Example 7. Example of Reactor code with timeout and fallback
Example 8. Example of CompletableFuture combination
Example 9. Example of Reactor code equivalent to future code
Example 11. Example of state-based generate
```

So, now, for these titles, they are rendered without the `Example N`prefix, but we still see the `Example` word, because these few examples title are starting with the `Example` word:

```
Example of Callback Hell
Example of Reactor code equivalent to callback code
Example of Reactor code with timeout and fallback
Example of CompletableFuture combination
Example of Reactor code equivalent to future code
Example of state-based generate
```

> The other problem is that the rendering of these "Example" headers don't provide a background for the monospace font and it is inconsistent with what we had and what other monospaced highlights provide.

I suspect that this is also related to the antora-ui-projectreactor (I'm not sure).

Since for the moment, I'm unable to re-enable automatic generation of the `Example N.` prefix, I would prefer to address this issue in a specific new PR, if you don't mind.

Reply:
If you decide to make a change in additional PR, please create an issue so that we do not forget. Thanks!

Reply:
https://github.com/reactor/reactor-core/issues/3811